modules/alloy/src/main/ts/ephox/alloy/ui/schema/MenuSchema.ts

```diff
@@ -124,7 +124,8 @@ const schema = Fun.constant([
   FieldSchema.defaulted('fakeFocus', false),
   FieldSchema.defaulted('focusManager', FocusManagers.dom()),
   Fields.onHandler('onHighlight'),
-  Fields.onHandler('onDehighlight')
+  Fields.onHandler('onDehighlight'),
+  FieldSchema.defaulted('menuRole', true),
```

Comment:
We should add an alloy changelog entry as well

Reply:
done